roms_tools/tests/test_setup/conftest.py

```diff
@@ -18,6 +18,9 @@ def pytest_addoption(parser):
         default=[],
         help="Specify which fixtures to overwrite. Use 'all' to overwrite all fixtures.",
     )
+    parser.addoption(
+        "--use_dask", action="store_true", default=False, help="Run tests with Dask"
+    )
```

Comment:
It would be good to run the tests once with and once without dask, in different CI jobs.

Reply:
Good idea. As of [f3e9c5e](https://github.com/CWorthy-ocean/roms-tools/pull/121/commits/f3e9c5e80aef826bd45911485c6f113c98b8e174), the tests are run with and without dask in the CI. This doubles the CI test suite run time.

Reply:
Are these getting run in the same CI jobs? I don't see them in the list below. Having a separately listed job would be useful for quickly diagnosing the cause of a failure I think.

We could ask `--use-dask` to the matrix, and test every combination of python version with and without dask. But that seems overkill. Perhaps we should run most jobs without (with?) dask, and one one extra job with (without?) dask.

Reply:
They are being run within the same CI job.
```
      - name: Running Tests (without Dask)
        shell: bash -l {0}
        run: |
           python -V
           coverage run --rcfile=coverage.toml -m pytest --verbose roms_tools/tests/*
      - name: Running Tests (with Dask)
        shell: bash -l {0}
        run: |
           python -V
           coverage run --rcfile=coverage.toml -m pytest --use_dask --verbose roms_tools/tests/*
```
but I can make a separate job.

Reply:
Done! The CI tests now include:
* 3.10-build-without-dask
* 3.11-build-without-dask
* 3.12-build-without-dask
* 3.12-build-with-dask

Reply:
Amazing, thanks!